Dockerfile

```diff
@@ -34,12 +33,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     echo 'LANGUAGE="en_US:en"' >> /etc/default/locale && \
     locale-gen en_US.UTF-8 && \
     update-locale en_US.UTF-8 && \
-    apt-key adv --fetch-keys https://packages.erlang-solutions.com/ubuntu/erlang_solutions.asc && \
     add-apt-repository -y ppa:ondrej/php && \
-    add-apt-repository -y ppa:openjdk-r/ppa && \
-    add-apt-repository -y ppa:git-core/ppa && \
-    add-apt-repository -y ppa:deadsnakes/ppa && \
```

Comment:
Curious about this 2 repositories being removed. I see Erlang was moved, but curious about why removing git-code and deadsnakes (what even is that?)

Reply:
it seemed to make no difference so I removed them. The repositories are only used for apt-get installs and I think they where added for erlang.

As they are not used anymore (and the dockerfile is the only place for apt-get install) we should be safe to remove them.

Reply:
deadsnakes packages python versions for ubuntu which historically lagged behind in the default repos of certain releases I believe - https://launchpad.net/%7Edeadsnakes/+archive/ubuntu/ppa. I don't think it's needed for focal any longer given the versions we're installing:
- https://packages.ubuntu.com/focal/python/

As for `git` though - https://launchpad.net/~git-core/+archive/ubuntu/ppa - we should double check the version we're installing by dropping this ppa (and compare to what we have right now). In the long run I also think we should probably pin this since it's a critical dep for our whole build pipeline.

Reply:
added the git core back for now but still don't think we need it as it is just the repository (and we are not doing anything with it.